### PR TITLE
Responsive web design project: thenextweb.com clone

### PR DESCRIPTION
**Pull request by Zubenna**
# thenextweb.com clone project: Summary of work done

## In this project I did the following;

- Set up the detailed layout of the web page
- Arranged the sections in groups.
- Used CSS grid to arrange columns in each group.
- Used CSS flex-box to arrange each section further where necessary.
- Added background images where necessary.
- Applied linear-gradient on background images.
- Used media query to apply responsiveness at min-width: 320px, min-width: 768px, min-width: 1024px

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Zubenna  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- Let's fix the with of the rest of the section from the latest news section till the latest deal section,
- There should be some margin allowance as with the original site.
- This is what we currently have: 👇 
latest_news_clone
-This is what is expected: 👇 
latest_news
##
Please make the suggested changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by Zubenna** [comment]
I have implemented your recommendation by increasing the left and right margins. Although the first version appeared exactly like the original website on my computer. If need be, may be you can specify a particular width/resolution for us to consider. It looks very ok on my computer.
Thanks for the review.

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,1030 @@
...
+
+.footnote .stackpath-logo {
+  width: 5rem;
+  height: 3rem;
+  padding-bottom: 2.2rem;
+}
+
+/* media query: min-width: 320px */
+
+@media only screen and (min-width: 320px) {
```
Comment (men32z) [inline style/style.css]: this kind of comments for instance

**Review by men32z — APPROVED** [review]
## Good job making the changes, you can merge.
- [ ] before to merge remove comments in styles as the following code is descriptive

BlgS1sdIUAEybkW

### Status Approved

--Luis Preza

**Comment by Zubenna** [comment]
The project has been approved.

**Pull request merged**